routes/users.js

```diff
@@ -21,6 +22,7 @@ router.get("/:userId/intro", authenticate, authorizeRoles([SUPERUSER]), users.ge
 router.put("/self/intro", authenticate, userValidator.validateJoinData, users.addUserIntro);
 router.get("/:id/skills", users.getUserSkills);
 router.get("/:id/badges", getUserBadges);
+router.patch("/:userId/roles", authenticate, authorizeRoles([SUPERUSER]), updateRoles);
```

Comment:
in the route to stay consistent with other routes we can use `id`in place of `userId`

Reply:
If you see there is also used :userId in routes/user.js file -> https://github.com/Real-Dev-Squad/website-backend/blob/develop/routes/users.js#L20

Reply:
Done